pkg/install/v1alpha1/version.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	mver "github.com/openebs/maya/pkg/version"
+)
+
+type version string
+
+const (
+	version070     version = "0.7.0"
+	invalidVersion version = "invalid.version"
+)
+
+func CurrentVersion() version {
+	return version(mver.GetVersion())
+}
+
+func Version(version string) version {
```

Comment:
@AmitKumarDas - Can we add an unit test that fails if an invalid version is returned? 

When a version file is changed and if the entry for the version is not updated in this file, we could get into a situation where CurrentVersion returns invalid. Adding this UT will make sure that travis will fail, if only version file is changed and not this code.